spec/test/memory_adapter_lint_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'rom/adapter/memory'
+require 'rom/adapter/lint/test'
+
+require 'minitest/autorun'
+
+class MemoryAdapterLintTest < MiniTest::Test
```

Comment:
I saw your changes to the constant. There's something else to keep in mind, Minitest 5 is `Minitest::Test` <- note the 't'. I believe v5.x includes an alias so `MiniTest` will still work.